modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -132,6 +132,68 @@ users filter ((item, index) -> item.error == null)
 ]
 ----
 
+=== Example
+
+This example reads a JSON array and uses the `filter` function to extract the phone numbers which are active.
+
+==== Input
+[source,JSON, linenums]
+----
+{
+  "Id": "1184001100000000517",
+  "marketCode": "US",
+  "languageCode": "en-US",
+  "profile": {
+  "base": {
+     "username": "TheMule",
+     "activeInd": "R",
+     "phone": [
+       {
+          "activeInd": "Y",
+          "type": "mobile",
+          "primaryInd": "Y",
+          "number": "230678123"
+       },
+       {
+         "activeInd": "N",
+         "type": "mobile",
+         "primaryInd": "N",
+         "number": ""
+       }
+      ]
+    }
+  }
+ }
+----
+
+==== DataWeave Script
+[source,DataWeave, linenums]
+----
+%dw 2.0
+output application/json
+---
+{
+    id: payload.Id,
+    markCode: payload.marketCode,
+    languageCode: payload.languageCode,
+    username: payload.profile.base.username,
+    phoneNumber: {(payload.profile.base.phone filter ($.activeInd == "Y" and $.primaryInd== "Y"))}.number default ""
```

Comment:
```suggestion
    phoneNumber: (payload.profile.base.phone filter ($.activeInd == "Y" and $.primaryInd== "Y")).number default []
```
Removed the `{}` because in that case if there is more than one active phone, this code will bring just the first one. It has more sense if it brings an array with all the phone numbers.